src/mmw/js/src/modeling/models.js

```diff
@@ -652,6 +656,9 @@ var ScenarioModel = Backbone.Model.extend({
 
                 pollSuccess: function() {
                     self.setResults();
+                    if (self.waitToRefresh) {
+                        self.get('results').trigger('reset');
```

Comment:
Doing a reset there without the silent option set to true will definitely do a reset, but it also has other side effects (including emptying the existing collection, I believe).